include/hpcombi/perm16.hpp

```diff
@@ -145,7 +154,9 @@ struct Transf16 : public PTransf16 {
     explicit operator uint64_t() const;
 };
 
-//! Partial permutation of @f$\{0, \dots, 15\}@f$
+/** Partial permutation of @f$\{0\dots 15\}@f$; see HPCombi::Perm16;
```

Comment:
```suggestion
/** Partial permutation of @f$\{0\dots 15\}@f$; see HPCombi::PPerm16;
```
?

Reply:
The doc of PPerm suggest to have a look at Perm. Rewritten as
see also HPCombi::Perm16;